src/backend/main_window.ts

```diff
@@ -24,6 +25,7 @@ export const sendFrontendMessage = (message: string, ...payload: unknown[]) => {
   }
 
   mainWindow.webContents.send(message, ...payload)
+  backendEvents.emit(message, ...payload)
```

Comment:
Maybe instead of adding this here in the main window's sendFrontendMessage, we could have some functions `gameStatusChange` and `installProgressUpdate` somewhere else that makes more semantic sense (not sure, somewhere in the store managers?)

Then you can replace all the `sendFrontendMessage('gameStatusUpdate'...` with `gameStatusChange(....` and similar for the `sendFrontendMessage(progressUpdate-...`

then those functions could call both sendFrontendMessage and backendEvents.emit

I really think the `sendFrontendMessage` shouldn't have mixed logic of sending a webContents message and emitting a backend event

Reply:
Okay, I pushed an update that uses helper methods for these events instead, LMK what you think.